websites/A/Autocode/presence.ts

```diff
@@ -0,0 +1,45 @@
+const presence = new Presence({
+clientId: "858292108195921920"
+}), supportedLanguages: Array<string> = [ "js", "md", "json", "gitignore", "txt", "html", "css"];
```

Comment:
You don't really have to give it a type here, just a comment no change is required